### PR TITLE
fix: upgrade ses to 0.12.3 to avoid console noise

### LINKED ISSUE #2324: get rid of <[Object: null prototype] {}> console noise
After applying both https://github.com/Agoric/agoric-sdk/pull/2303 and https://github.com/Agoric/SES-shim/pull/569 , a `yarn test` fails in something having to do with xsnap, complaining about an assignment to a `constructor` property failing.

***Low priority*** because https://github.com/Agoric/SES-shim/pull/569 is low priority. But fixing this would cause the Node console to stop adding junk like the following to its display of objects.
```js
[Function (anonymous)] Function <Function <[Object: null prototype] {}>>
```

### PR DESCRIPTION
**Pull request by dckc**
fixes #2324

### CODE REVIEW

**Review by erights — APPROVED** [review]
LGTM!

**Comment by dckc** [comment]
lint:types is not happy. interesting!

```
> @agoric/swingset-vat
$ yarn lint:types&&yarn lint:eslint
$ tsc -p jsconfig.json
Error: ../notifier/src/notifier.js(90,30): error TS2345: Argument of type 'UpdateRecord<T> | undefined' is not assignable to parameter of type 'UpdateRecord<T> | PromiseLike<UpdateRecord<T>>'.
```

**Review by michaelfig — APPROVED** [review]
LGTM!

**Comment by dckc** [comment]
@FUDCo any clues on the swingset-runner test/test-demo.js:25 failure?

https://github.com/Agoric/agoric-sdk/pull/2552/checks?check_run_id=2009422750

File: packages/swingset-runner/bin/runner [hunk 1]
```diff
@@ -6,7 +6,20 @@
...
+// LMDB bindings need to be imported before lockdown.
+import 'node-lmdb';
+
+// FIXME: Without importing vega ahead of lockdown,
+// node -r esm bin/runner --init --memdb run demo/encouragementBot
+// exits with error code 1, just printing `{}`.
+//
+// Since this started with ses@0.12.3 we suspect vega is somehow
+// hitting the override mistake with `constructor`.
+import 'vega';
```
Comment (michaelfig) [inline packages/swingset-runner/bin/runner:19]: We need to investigate this.

Comment (FUDCo) [reply packages/swingset-runner/bin/runner:19]: FYI, the comment is too specific.  The problem will occur with any invocation of swingset-runner regardless of the command line options.

Comment (erights) [reply packages/swingset-runner/bin/runner:19]: See https://github.com/Agoric/agoric-sdk/pull/2575

**Pull request merged**